### ISSUE TITLE
Feature: floppy activity LED and track display

### ISSUE THREAD

**Issue by darkstar** [jeffpar/pcjs.v1#16]
How about a simple LED-style box next to the floppy and optionally a textbox which shows at what track the drive is? This would make waiting for the OS/2 installation a bit more fun :)

**Comment by jeffpar** [comment]
Agreed. In the meantime though, I wouldn't recommend trying to install OS/2 until I track down whatever's causing protected-mode hangs. I was happy to get OS/2 working at all, but obviously there's a bit more work to do before it's ready for prime time.

**Comment by Foxlum** [comment]
This seems to be a forgotten feature request. i'm not sure if you are still working on this @jeffpar or you just forgot.
